main_script.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import csv
+from constants import (
+    PERFORMANCE_REWARDS_BUDGET,
+    CONSISTENCY_REWARDS_BUDGET,
+    QUOTE_REWARDS_BUDGET,
+    SOLVER_ADDRESSES,
+)
+from gnosis_scan_api import get_block_range
+from execute_sql_queries import (
+    get_auction_range,
+    compute_solver_rewards,
+    compute_quote_rewards,
+    execute_participation_rewards_helper,
+)
+
+
+def main() -> None:
+    year = int(sys.argv[1])
+    month = int(sys.argv[2])
+    day = int(sys.argv[3])
+    start_block, end_block = get_block_range(year, month, day)
+    print(
+        "\nAccounting period from block "
+        + str(start_block)
+        + " until block "
+        + str(end_block)
+        + "."
+    )
+
+    ## Compute prod and barn auction range
+    (
+        prod_start_auction_str,
+        prod_end_auction_str,
+        barn_start_auction_str,
+        barn_end_auction_str,
+    ) = get_auction_range(str(start_block), str(end_block))
+
+    print(
+        "Production auctions from "
+        + prod_start_auction_str
+        + " until "
+        + prod_end_auction_str
+        + ".\n"
+        + "Barn auctions from "
+        + barn_start_auction_str
+        + " until "
+        + barn_end_auction_str
+        + ".\n"
+    )
+
+    FINAL_CONSISTENCY_REWARDS_BUDGET = CONSISTENCY_REWARDS_BUDGET
+    # computing solver performance rewards
+    performance_rewards_per_solver = {}
+    performance_spend = 0
+    performance_rewards = compute_solver_rewards(str(start_block), str(end_block))
+    for index, row in performance_rewards.iterrows():
+        amount = int(row["primary_reward_xdai"])
+        if amount > 0:
+            performance_spend += amount
+        performance_rewards_per_solver[row["solver"]] = amount
+
+    for solver in performance_rewards_per_solver:
+        if performance_rewards_per_solver[solver] > 0:
+            performance_rewards_per_solver[solver] = (
+                performance_rewards_per_solver[solver]
+                * PERFORMANCE_REWARDS_BUDGET
+                / performance_spend
+            )
+        else:
+            FINAL_CONSISTENCY_REWARDS_BUDGET += (-1) * performance_rewards_per_solver[solver]
+
+    # computing quote rewards
+    quote_rewards_per_solver = {}
+    total_quotes = 0
+    quote_rewards = compute_quote_rewards(str(start_block), str(end_block))
+    for index, row in quote_rewards.iterrows():
+        number = int(row["num_quotes"])
+        if number > 0:
+            total_quotes += number
+            quote_rewards_per_solver[row["solver"]] = number
+
+    for solver in quote_rewards_per_solver:
+        quote_rewards_per_solver[solver] = (
+            quote_rewards_per_solver[solver] * QUOTE_REWARDS_BUDGET / total_quotes
+        )
+
+    # computing consistency rewards
+    participation_data = execute_participation_rewards_helper(
+        str(start_block), str(end_block)
+    )
+    participation_per_solver = {}
+    total_participation_tokens = 0
```

Comment:
To be more constent with the loop above, maybe use `consistency_spend` (or `consistency_spent`?).
```suggestion
    consistency_spend = 0
```